docker-compose.yml

```diff
@@ -23,15 +23,15 @@ version: "3.9"
 
 services:
   db:
-    image: couchbase/server-sandbox:7.0.0
+    image: couchbase/server-sandbox:7.1.0
     ports:
-      - "8091-8095:8091-8095"
+      - "8000-8005:8000-8005"
       - "9102:9102"
-      - "11210:11210"
+      - "11200:11200"
     expose:
-      - "8091"
-      - "8094"
-    container_name: couchbase-sandbox-7.0.0
+      - "8001"
+      - "8004"
+    container_name: couchbase-sandbox-7.1.0
```

Comment:
It would be better to make a 7.1 branch (for both 7.1 server, and current Capella).
This will also leave the 7.0 branch working as-is for all existing users, and those steered there by the docs.

Reply:
This way we could iteratively address issues as they arise and improve the branch for both Capella and Couchbase.